protocols/cosi/packets.go

```diff
@@ -15,6 +15,17 @@ import (
 type CosiAnnouncement struct {
 	// From = TreeNodeId in the Tree
 	From uuid.UUID
+	// Tells to the node whether they should make the verification or not
+	// XXX XXX UGLY HACK XXX XXX
+	// Is to be removed during the next versions of SDA. Need that for the
+	// moment because no way to have external configuration for the intermediate
+	// and leaf nodes.
+	// see https://github.com/dedis/cothority/issues/260
+	// 0 - no check at all
+	// 1 - check only at root
+	// 2 - check at each level of the tree
+	VerifyResponse int
+
```

Comment:
Please, make an example because I don't see how can I access each node in this function.
Moreover, when we will attack the external api issue, a lot of things will be able to pass through that.
Anyway, it's only for simulation anyway, all theses protocols should *not* be used as something else...!